src/client/driver/iframe-driver.js

```diff
@@ -80,8 +80,11 @@ export default class IframeDriver extends Driver {
                 var inCommandExecution = this.contextStorage.getItem(this.COMMAND_EXECUTING_FLAG) ||
                                          this.contextStorage.getItem(this.EXECUTING_IN_IFRAME_FLAG);
 
-                if (inCommandExecution)
+                if (inCommandExecution) {
```

Comment:
In top window we set flags to `false` in any way, but in iframe only in case of `inCommandExecution === true`. May be we should do it in iframe the same?